chapters/synchronous.tex

```diff
@@ -1400,13 +1400,13 @@ \subsection{Inferencing of solverMethod}\label{inferencing-of-solvermethod}
 
 \section{Initialization of Clocked Partitions}\label{initialization-of-clocked-partitions}
 
-The standard scheme for initialization of Modelica models does not apply for clocked base-partitions.
+The standard scheme for initialization of Modelica models does not apply for discrete-time sub-partitions.
 Instead, initialization is performed in the following way:
 \begin{itemize}
 \item
-  Variables in clocked partitions cannot be used in initial equation or initial algorithm sections.
+  Variables in discrete-time sub-partitions cannot be used in initial equation or initial algorithm sections.
 \item
-  Attribute \lstinline!fixed! cannot be applied to variables in clocked partitions.
+  Attribute \lstinline!fixed! cannot be applied to variables in discrete-time sub-partitions.
   The attribute \lstinline!fixed! is true for variables to which \lstinline!previous! is applied, otherwise false.
```

Comment:
While we are at it, shouldn't we avoid the problem when `previous` is applied to a parameter or constant?
```suggestion
  The attribute \lstinline!fixed! is true for clocked states, otherwise false.
```